R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -971,10 +971,15 @@ test_that("spark.randomForest Classification", {
   predictions <- collect(predict(model, data))$prediction
   expect_equal(length(grep("1.0", predictions)), 50)
   expect_equal(length(grep("2.0", predictions)), 50)
+
+  # spark.randomForest classification can work on libsvm data
+  data <- read.df(absoluteSparkPath("data/mllib/sample_multiclass_classification_data.txt"),
+                source = "libsvm")
+  model <- spark.randomForest(data, label ~ features, "classification")
+  expect_equal(summary(model)$numFeatures, 4)
 })
 
-test_that("spark.gbt", {
-  # regression
+test_that("spark.gbt regression", {
```

Comment:
I don't have strong preference as well, I'll merge random forest tests into one as you suggested.